packages/vapor/scss/tables/table.scss

```diff
@@ -157,6 +157,10 @@
             position: relative;
             top: -1px; // Small hack to make it work in all browsers...
         }
+
+        &.coveo-checkbox-width {
+            width: 142px;
```

Comment:
I'm kind of confused about this 142px, it seems to be the width of the label in the table and it has nothing to do with checkboxes

![image](https://user-images.githubusercontent.com/260007/63268765-ee2cae80-c262-11e9-8bc2-bb7d8decc511.png)

IMHO this style won't be shared with any other tables, thus it should go in the admin

Reply:
if we want to add this classes, we need to remove the coveo- before, nothing related to coveo should be added in react-vapor